assets/js/modules/analytics/components/setup/SetupForm.test.js

```diff
@@ -112,6 +112,9 @@ describe( 'SetupForm', () => {
 			);
 
 		await registry.dispatch( MODULES_ANALYTICS ).selectAccount( accountID );
+		await registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.selectAccount( accountID );
```

Comment:
I think this is not necessary here. These tests will be refactored when this component is recreated for `analytics-4` as part of #7929, where this action can be dispatched only for `analytics-4` instead of `analytics`.